client/src/components/PacmanGame/entities/Pacman.jsx

```diff
@@ -1,14 +1,29 @@
 import React from 'react';
 import { Rect } from 'react-konva';
 
-const Pacman = ({ x, y, gridSize }) => (
-  <Rect
-    x={x * gridSize}
-    y={y * gridSize}
-    width={gridSize}
-    height={gridSize}
-    fill="yellow"
-  />
-);
+const KEYS = {left: 37, up: 38, right: 39, down: 40};
+
+class Pacman extends React.Component {
+  constructor(props) {
+    super(props) 
+    this.state = {
+      direction: KEYS.right,
+      x: 0,
+      y: 0
+    }
+  }
+
+  render () {
+    return (
+           <Rect
+                x={this.props.x * this.props.gridSize}
```

Comment:
Destructure the props.

Reply:
Pacman.js is still WIP. Will refactor it in the next commit with event listeners and few methods

Reply:
Why are we merging it then. We can complete this part and then merge.